docs/app/documentation/component-docs/split-button/split-button-docs.component.html

```diff
@@ -0,0 +1,55 @@
+<h2>Button Options</h2>
+<description>
+  There are three emphasis styles used to indicate the importance of the button on the page. Do not provide a value for
+  the default option. Use <code class="code-snippet">[options]="'emphasized'"</code> or
+  <code class="code-snippet">[options]="'light'"</code> to set the option.
+</description>
+<component-example [name]="'ex0'">
+    <fd-split-button-options-example></fd-split-button-options-example>
+</component-example>
+<code-example [code]="buttonSplitOptionsExample" [language]="'HTML'"></code-example>
+
+<separator></separator>
+<h2>Button Types</h2>
+<description>
+  The default button type is "action", no modifier is required. Use
+  <code class="code-snippet">[fdType]="'standard'", [fdType]="'positive'", [fdType]="'medium'"</code>, or
+  <code class="code-snippet">[fdType]="'negative'"</code> to modify the type of the button.
+</description>
+<component-example [name]="'ex1'">
+    <fd-split-button-types-example></fd-split-button-types-example>
+</component-example>
+<code-example [code]="buttonHtmlType" [language]="'HTML'"></code-example>
+
+<separator></separator>
+<h2>Buttons with Icons</h2>
+<description>
+  Use
+  <code class="code-snippet">[glyph]="'{{ '{' }}icon-name}'"</code> to change an icon of dropdown menu.
+</description>
+<component-example [name]="'ex3'">
+  <fd-split-button-icons-example></fd-split-button-icons-example>
+</component-example>
+<code-example [code]="buttonSplitIcons" [language]="'HTML'"></code-example>
+
+<separator></separator>
+<h2>Open/Close programmatically</h2>
+<description>
+    There is possibility to control open/close of popover by 2-ways data binding <code>[(isOpen)]</code>
```

Comment:
It is possible to control the open state of the popover through two-way binding to the isOpen property.